doc/developer/building-frr-for-debian12.rst

```diff
@@ -0,0 +1,118 @@
+Debian 12
+=========
+
+Install required packages
+-------------------------
+
+Add packages:
+
+::
```

Comment:
A new line is missing here after `::`. Which in turn formats the highlighted text wrongly.

Reply:
Ah this one!